src/main/java/calculator/Calculator.java

```diff
@@ -0,0 +1,63 @@
+package calculator;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Calculator {
+    private static final String CUSTOM_REGEX = "//(.)\n(.*)";
+    private static final String STANDARD_SPLIT_REGEX = "[,:]";
+    private static final int DELIMETER_INDEX = 1;
```

Comment:
```suggestion
    private static final int DELIMITER_INDEX = 1;
```
저도 많은 오타를 내는 부분인데요 ㅎ_ㅎ..
다음엔 저와 함께 오타를 안내는 방향으로 해보아요.